### ISSUE TITLE
Add Facebook event page so that folks can share FB event.

### ISSUE THREAD

**Issue by seabre** [openchattanooga/citycampcha#4]

**Comment by aplannersguide** [comment]
The event page exists at https://www.facebook.com/events/317720138436891/. Is this issue about adding a link on the citycampcha.com website to the facebook page? Like a "share with a friend" type thing?

**Comment by seabre** [comment]
Yes this is about adding a link to the citycampcha.com page so folks can share.
